src/components/Month.tsx

```diff
@@ -23,7 +23,7 @@ const Month = (props: { className?: string; index: number; title: string; year:
   return (
     <div className={`${props.className} ${monthClasses}`}>
       <div className="font-bold">{props.title}</div>
```

Comment:
Add "cursor-default select-none" - currently we are seeing cursor-text when the mouse hovers over the text